src/api/products.json

```diff
@@ -0,0 +1,263 @@
+[
```

Comment:
Use https://mate-academy.github.io/react_phone-catalog/api/products.json to fetch data